robot/Cumulus/resources/locators_48.py

```diff
@@ -250,6 +250,14 @@
     'modal-form':{
         'label':'//div[./*/*[text()="{}"]]',
         },
+    'gift_entry':{
+        'button':'//*[@data-qa-locator="button {}"]/button',
+        'field_input':'//*[contains(@data-qa-locator,"{}")]//child::input',
+        'textarea':'//*[contains(@data-qa-locator,"{}")]//child::textarea',
```

Comment:
Can we pass the tags directly in the function/keyword where it getting called, since the only difference between the locator is the tag name?

Reply:
didn't think about that, thanks :)

Reply:
made change